### PR TITLE
init template user and add file css js

### PR DESCRIPTION
**Pull request by minhvip1997**
Tạo trang home page.
Tạo trang list book, detail book.
Tạo trang crud review.
Tạo trang crud comment.
Tạo trang cart và checkout.
Tạo trang hồ sơ người dùng , timeline history, rate, favorite

### CODE REVIEW

File: rikai/public/js/plugins2.js [hunk 1]
```diff
@@ -0,0 +1,2220 @@
...
+					13 : 'left', // enter
+					34 : 'up',   // page down
+					39 : 'left', // right arrow
+					40 : 'up'    // down arrow
+				},
+				prev : {
+					8  : 'right',  // backspace
+					33 : 'down',   // page up
+					37 : 'right',  // left arrow
+					38 : 'down'    // up arrow
```
Comment (chien-rikai) [inline rikai/public/js/plugins2.js:117]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rikai-academy/book_system_01/pullRequest?prid=7834536)
```suggestion
					38 : "down"    // up arrow
```

File: rikai/public/js/custom.js [hunk 2]
```diff
@@ -0,0 +1,568 @@
...
+
+			if(!$parent.parent().hasClass('nav')) {
+				$el.next().css({"top": $el[0].offsetTop, "left": $parent.outerWidth() - 4});
+			}
+			$open.not($(this).parents("li")).removeClass("open");
+			return false;
+		});
+	}
+	//js for nav icon 
+	var clickMenubtn = $('#nav-icon1');
```
Comment (chien-rikai) [inline rikai/public/js/custom.js:46]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rikai-academy/book_system_01/pullRequest?prid=7834536)
```suggestion
	var clickMenubtn = $("#nav-icon1");
```

File: rikai/public/js/plugins2.js [hunk 3]
```diff
@@ -0,0 +1,2220 @@
...
+				},
+				close  : [27], // escape key
+				play   : [32], // space - start/stop slideshow
+				toggle : [70]  // letter "f" - toggle fullscreen
+			},
+
+			direction : {
+				next : 'left',
```
Comment (chien-rikai) [inline rikai/public/js/plugins2.js:125]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rikai-academy/book_system_01/pullRequest?prid=7834536)
```suggestion
				next : "left",
```

File: rikai/public/js/plugins2.js [hunk 4]
```diff
@@ -0,0 +1,2220 @@
...
+				error    : '<p class="fancybox-error">The requested content cannot be loaded.<br/>Please try again later.</p>',
+				closeBtn : '<a title="Close" class="fancybox-item fancybox-close" href="javascript:;"></a>',
+				next     : '<a title="Next" class="fancybox-nav fancybox-next" href="javascript:;"><span></span></a>',
+				prev     : '<a title="Previous" class="fancybox-nav fancybox-prev" href="javascript:;"><span></span></a>',
+				loading  : '<div id="fancybox-loading"><div></div></div>'
+			},
+
+			// Properties for each animation type
+			// Opening fancyBox
+			openEffect  : 'fade', // 'elastic', 'fade' or 'none'
```
Comment (chien-rikai) [inline rikai/public/js/plugins2.js:152]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rikai-academy/book_system_01/pullRequest?prid=7834536)
```suggestion
			openEffect  : "fade", // 'elastic', 'fade' or 'none'
```

File: rikai/public/js/plugins2.js [hunk 5]
```diff
@@ -0,0 +1,2220 @@
...
+			closeMethod  : 'zoomOut',
+
+			// Changing next gallery item
+			nextEffect : 'elastic', // 'elastic', 'fade' or 'none'
+			nextSpeed  : 250,
+			nextEasing : 'swing',
+			nextMethod : 'changeIn',
+
+			// Changing previous gallery item
+			prevEffect : 'elastic', // 'elastic', 'fade' or 'none'
```
Comment (chien-rikai) [inline rikai/public/js/plugins2.js:172]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rikai-academy/book_system_01/pullRequest?prid=7834536)
```suggestion
			prevEffect : "elastic", // 'elastic', 'fade' or 'none'
```

File: rikai/public/js/plugins2.js [hunk 6]
```diff
@@ -0,0 +1,2220 @@
...
+			closeEffect  : 'fade', // 'elastic', 'fade' or 'none'
+			closeSpeed   : 250,
+			closeEasing  : 'swing',
+			closeOpacity : true,
+			closeMethod  : 'zoomOut',
+
+			// Changing next gallery item
+			nextEffect : 'elastic', // 'elastic', 'fade' or 'none'
+			nextSpeed  : 250,
+			nextEasing : 'swing',
```
Comment (chien-rikai) [inline rikai/public/js/plugins2.js:168]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rikai-academy/book_system_01/pullRequest?prid=7834536)
```suggestion
			nextEasing : "swing",
```

File: rikai/public/js/plugins2.js [hunk 7]
```diff
@@ -0,0 +1,2220 @@
...
+				loading  : '<div id="fancybox-loading"><div></div></div>'
+			},
+
+			// Properties for each animation type
+			// Opening fancyBox
+			openEffect  : 'fade', // 'elastic', 'fade' or 'none'
+			openSpeed   : 250,
+			openEasing  : 'swing',
+			openOpacity : true,
+			openMethod  : 'zoomIn',
```
Comment (chien-rikai) [inline rikai/public/js/plugins2.js:156]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rikai-academy/book_system_01/pullRequest?prid=7834536)
```suggestion
			openMethod  : "zoomIn",
```

File: rikai/public/js/plugins2.js [hunk 8]
```diff
@@ -0,0 +1,2220 @@
...
+				next     : '<a title="Next" class="fancybox-nav fancybox-next" href="javascript:;"><span></span></a>',
+				prev     : '<a title="Previous" class="fancybox-nav fancybox-prev" href="javascript:;"><span></span></a>',
+				loading  : '<div id="fancybox-loading"><div></div></div>'
+			},
+
+			// Properties for each animation type
+			// Opening fancyBox
+			openEffect  : 'fade', // 'elastic', 'fade' or 'none'
+			openSpeed   : 250,
+			openEasing  : 'swing',
```
Comment (chien-rikai) [inline rikai/public/js/plugins2.js:154]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rikai-academy/book_system_01/pullRequest?prid=7834536)
```suggestion
			openEasing  : "swing",
```

File: rikai/public/js/plugins2.js [hunk 9]
```diff
@@ -0,0 +1,2220 @@
...
+			index   : 0,
+			type    : null,
+			href    : null,
+			content : null,
+			title   : null,
+
+			// HTML templates
+			tpl: {
+				wrap     : '<div class="fancybox-wrap" tabIndex="-1"><div class="fancybox-skin"><div class="fancybox-outer"><div class="fancybox-inner"></div></div></div></div>',
+				image    : '<img class="fancybox-image" src="{href}" alt="" />',
```
Comment (chien-rikai) [inline rikai/public/js/plugins2.js:141]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rikai-academy/book_system_01/pullRequest?prid=7834536)
```suggestion
				image    : "<img class=\"fancybox-image\" src=\"{href}\" alt=\"\" />",
```

File: rikai/public/js/custom.js [hunk 10]
```diff
@@ -0,0 +1,568 @@
+//preloading for page
+$(window).on('load', function() { // makes sure the whole site is loaded
```
Comment (chien-rikai) [inline rikai/public/js/custom.js:2]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rikai-academy/book_system_01/pullRequest?prid=7834536)
```suggestion
$(window).on("load", function() { // makes sure the whole site is loaded 
```

File: rikai/public/js/plugins2.js [hunk 11]
```diff
@@ -0,0 +1,2220 @@
...
+			swf : {
+				wmode: 'transparent',
+				allowfullscreen   : 'true',
+				allowscriptaccess : 'always'
+			},
+
+			keys  : {
+				next : {
+					13 : 'left', // enter
+					34 : 'up',   // page down
```
Comment (chien-rikai) [inline rikai/public/js/plugins2.js:109]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rikai-academy/book_system_01/pullRequest?prid=7834536)
```suggestion
					34 : "up",   // page down
```

File: rikai/public/js/plugins2.js [hunk 12]
```diff
@@ -0,0 +1,2220 @@
...
+			},
+			swf : {
+				wmode: 'transparent',
+				allowfullscreen   : 'true',
+				allowscriptaccess : 'always'
+			},
+
+			keys  : {
+				next : {
+					13 : 'left', // enter
```
Comment (chien-rikai) [inline rikai/public/js/plugins2.js:108]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rikai-academy/book_system_01/pullRequest?prid=7834536)
```suggestion
					13 : "left", // enter
```

File: rikai/public/js/plugins2.js [hunk 13]
```diff
@@ -0,0 +1,2220 @@
...
+			// Properties for each animation type
+			// Opening fancyBox
+			openEffect  : 'fade', // 'elastic', 'fade' or 'none'
+			openSpeed   : 250,
+			openEasing  : 'swing',
+			openOpacity : true,
+			openMethod  : 'zoomIn',
+
+			// Closing fancyBox
+			closeEffect  : 'fade', // 'elastic', 'fade' or 'none'
```
Comment (chien-rikai) [inline rikai/public/js/plugins2.js:159]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rikai-academy/book_system_01/pullRequest?prid=7834536)
```suggestion
			closeEffect  : "fade", // 'elastic', 'fade' or 'none'
```

File: rikai/public/js/custom.js [hunk 14]
```diff
@@ -0,0 +1,568 @@
+//preloading for page
+$(window).on('load', function() { // makes sure the whole site is loaded 
+	var status = $('#status');
+	var preloader = $('#preloader');
+	var body = $('body');
+	status.fadeOut(); // will first fade out the loading animation 
+	preloader.delay(0).fadeOut('fast'); // will fade out the white DIV that covers the website. 
+	body.delay(0).css({'overflow':'visible'});
```
Comment (chien-rikai) [inline rikai/public/js/custom.js:8]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rikai-academy/book_system_01/pullRequest?prid=7834536)
```suggestion
	body.delay(0).css({"overflow":'visible'});
```

File: rikai/resources/views/users/layouts/index.blade.php [hunk 15]
```diff
@@ -0,0 +1,20 @@
...
+
+<!--end of latest new v1 section-->
+<!-- footer section-->
+@include('users.layouts.footer')
+<!-- end of footer section-->
+
+<script src="js/jquery.js"></script>
+<script src="js/plugins.js"></script>
+<script src="js/plugins2.js"></script>
+<script src="js/custom.js"></script>
```
Comment (namtprikai) [inline rikai/resources/views/users/layouts/index.blade.php:15]: Tách ra 1 file blade riêng

File: rikai/resources/views/users/layouts/slide.blade.php [hunk 16]
```diff
@@ -0,0 +1,123 @@
...
+	    				<div class="cate">
+	    					<span class="green"><a href="#">comedy</a></span>
+	    				</div>
+	    				<h6><a href="#">Die hard</a></h6>
+	    				<p><i class="ion-android-star"></i><span>7.4</span> /10</p>
+	    			</div>
+	    		</div>
+	    	</div>
+	    </div>
+	</div>
```
Comment (namtprikai) [inline rikai/resources/views/users/layouts/slide.blade.php:122]: Check lại lỗi indent, Dùng thêm i18n

File: rikai/resources/views/users/profile/index.blade.php [hunk 17]
```diff
@@ -0,0 +1,122 @@
...
+								<input class="submit" type="submit" value="change">
+							</div>
+						</div>	
+					</form>
+				</div>
+			</div>
+		</div>
+	</div>
+</div>
+@endsection
```
Comment (namtprikai) [inline rikai/resources/views/users/profile/index.blade.php:122]: Sử dụng i18n, check lại lỗi indent.
Các trang khác cũng tương tự

File: rikai/resources/views/users/review/create.blade.php [hunk 18]
```diff
@@ -0,0 +1,98 @@
...
+								<input class="submit" type="submit" value="save">
+							</div>
+						</div>	
+					</form>
+				</div>
+			</div>
+		</div>
+	</div>
+</div>
+<script src="https://code.jquery.com/jquery-3.6.0.js" integrity="sha256-H+K7U5CnXl1h5ywQfKtSj8PCmoN9aaq30gDh27Xc0jk=" crossorigin="anonymous"></script>
```
Comment (namtprikai) [inline rikai/resources/views/users/review/create.blade.php:68]: download jquery về nhé. Không sử dụng link

File: rikai/resources/views/users/review/create.blade.php [hunk 19]
```diff
@@ -0,0 +1,98 @@
...
+			if(ratedIndex != 1)
+				for(var i=0; i<=ratedIndex; i++)
+					$('.ion-ios-star:eq('+i+')').css('color','yellow');
+		});
+	});
+
+	function resetStartColor(){
+		$('.ion-ios-star').css('color','#333')
+	}
+</script>
```
Comment (namtprikai) [inline rikai/resources/views/users/review/create.blade.php:97]: Tách ra 1 file script nhé

File: rikai/resources/views/users/review/edit.blade.php [hunk 20]
```diff
@@ -0,0 +1,101 @@
...
+			if(ratedIndex != 1)
+				for(var i=0; i<=ratedIndex; i++)
+					$('.ion-ios-star:eq('+i+')').css('color','yellow');
+		});
+	});
+
+	function resetStartColor(){
+		$('.ion-ios-star').css('color','#333')
+	}
+</script>
```
Comment (namtprikai) [inline rikai/resources/views/users/review/edit.blade.php:100]: Tách ra file script nhé

File: rikai/resources/views/users/review/show.blade.php [hunk 21]
```diff
@@ -0,0 +1,115 @@
...
+                     </form>
+                  </div>
+                  <!-- comment form -->
+               </div>
+            </div>
+         </div>
+      </div>
+   </div>
+</div>
+@endsection
```
Comment (namtprikai) [inline rikai/resources/views/users/review/show.blade.php:115]: Sử dụng i18n

File: rikai/routes/web.php [hunk 22]
```diff
@@ -14,5 +19,19 @@
...
+Route::get('bookdetail/{id}',[BookController1::class,'show']);
+Route::get('cart/{id}',[BookController1::class,'cart']);
+Route::get('checkout',[BookController1::class,'checkout']);
+Route::get('review/create',[ReviewController1::class,'create']);
+Route::get('review/show/{id}',[ReviewController1::class,'show']);
+Route::get('comment/edit/{id}',[CommentController1::class,'edit']);
+Route::get('profile',[ProfileController1::class,'index']);
+Route::get('profile/favoritebook/{id}',[ProfileController1::class,'favoriteBook']);
+Route::get('profile/ratebook/{id}',[ProfileController1::class,'rateBook']);
+Route::get('profile/timeline/{id}',[ProfileController1::class,'timeLine']);
```
Comment (namtprikai) [inline rikai/routes/web.php:37]: Sử dụng restful

File: rikai/resources/views/users/book/cart.blade.php [hunk 23]
```diff
@@ -0,0 +1,107 @@
+@extends('users.layouts.index')
+@section('content')
+<div class="hero common-hero">
+	<div class="container">
+		<div class="row">
+			<div class="col-md-12">
+				<div class="hero-ct">
+					<h1> movie listing - list</h1>
```
[the remark this answers is not in the document]
Comment (minhvip1997) [reply rikai/resources/views/users/book/cart.blade.php]: em đã fix

File: rikai/resources/views/users/book/cart.blade.php [hunk 24]
```diff
@@ -0,0 +1,107 @@
...
+</div>
+<div class="page-single movie_list">
+	<div class="container">
+		<div class="row ipad-width2">
+			<div class="col-md-12 col-sm-12 col-xs-12">
+                <table>
+                    <tr>
+                    <th class="book">Book</th>
+                    <th class="quantity">Quantity</th>
+                    <th class="subtotal">Subtotal</th>
```
[the remark this answers is not in the document]
Comment (minhvip1997) [reply rikai/resources/views/users/book/cart.blade.php]: em đã fix

File: rikai/resources/views/users/book/cart.blade.php [hunk 25]
```diff
@@ -0,0 +1,107 @@
...
+					        <div class="mv-item-infor">
+						    <h6><a href="moviesingle.html">oblivion <span>(2012)</span></a></h6>
+						    <p class="rate"><i class="ion-android-star"></i><span>8.1</span> /10</p>
+						    <p class="describe">Earth's mightiest heroes must come together and learn to fight as a team if they are to stop the mischievous Loki and his alien army from enslaving humanity...</p>
+						    <p>Director: <a href="#">Joss Whedon</a></p>
+					        </div>
+                            <br>
+                            <a href="#" class="remove-cart">Remove</a>
+				        </div>
+                        </td>
```
[the remark this answers is not in the document]
Comment (minhvip1997) [reply rikai/resources/views/users/book/cart.blade.php]: em đã fix

File: rikai/resources/views/users/book/cart.blade.php [hunk 26]
```diff
@@ -0,0 +1,107 @@
...
```
[the remark this answers is not in the document]
Comment (minhvip1997) [reply rikai/resources/views/users/book/cart.blade.php]: em đã fix

File: rikai/resources/views/users/book/checkout.blade.php [hunk 27]
```diff
@@ -0,0 +1,96 @@
...
+								<input type="text" placeholder="">
+							</div>
+							<div class="col-md-6 form-it">
+								<label>Cvv</label>
+								<input type="text" placeholder="">
+							</div>
+						</div>
+						<div class="row">
+							<div class="col-md-2">
+                            <button class="submit" type="submit" placeholder="update">Checkout</button>
```
[the remark this answers is not in the document]
Comment (minhvip1997) [reply rikai/resources/views/users/book/checkout.blade.php]: em đã fix

File: rikai/resources/views/users/book/checkout.blade.php [hunk 28]
```diff
@@ -0,0 +1,96 @@
...
+					<h1>Checkout</h1>
+					<ul class="breadcumb">
+						<li class="active"><a href="#">Home</a></li>
+						<li> <span class="ion-ios-arrow-right"></span>Checkout</li>
+					</ul>
+				</div>
+			</div>
+		</div>
+	</div>
+</div>
```
[the remark this answers is not in the document]
Comment (minhvip1997) [reply rikai/resources/views/users/book/checkout.blade.php]: em đã fix

File: rikai/resources/views/users/layouts/footer.blade.php [hunk 29]
```diff
@@ -0,0 +1,56 @@
...
+		</div>
+	</div>
+	<div class="ft-copyright">
+		<div class="ft-left">
+			<p><a target="_blank" href="https://www.templateshub.net">Templates Hub</a></p>
+		</div>
+		<div class="backtotop">
+			<p><a href="#" id="back-to-top">Back to top  <i class="ion-ios-arrow-thin-up"></i></a></p>
+		</div>
+	</div>
```
[the remark this answers is not in the document]
Comment (minhvip1997) [reply rikai/resources/views/users/layouts/footer.blade.php]: em đã fix

File: rikai/resources/views/users/layouts/header.blade.php [hunk 30]
```diff
@@ -0,0 +1,202 @@
...
+	<meta name=viewport content="width=device-width, initial-scale=1">
+	<meta name="format-detection" content="telephone-no">
+
+	<!-- CSS files -->
+	<link rel="stylesheet" href="css/plugins.css">
+	<link rel="stylesheet" href="css/style.css">
+	<base href="{{asset('')}}">
+</head>
+
+<body>
```
[the remark this answers is not in the document]
Comment (minhvip1997) [reply rikai/resources/views/users/layouts/header.blade.php]: em đã fix

File: rikai/resources/views/users/layouts/header.blade.php [hunk 31]
```diff
@@ -0,0 +1,202 @@
...
+        <div class="row">
+        	<p>Or via social</p>
+            <div class="social-btn-2">
+            	<a class="fb" href="#"><i class="ion-social-facebook"></i>Facebook</a>
+            	<a class="tw" href="#"><i class="ion-social-twitter"></i>twitter</a>
+            </div>
+        </div>
+    </div>
+</div>
+<!--end of login form popup-->
```
[the remark this answers is not in the document]
Comment (minhvip1997) [reply rikai/resources/views/users/layouts/header.blade.php]: em đã fix

File: rikai/resources/views/users/layouts/header.blade.php [hunk 32]
```diff
@@ -0,0 +1,202 @@
...
+                    <input type="password" name="password" id="repassword-2" placeholder="" pattern="(?=^.{8,}$)((?=.*\d)|(?=.*\W+))(?![.\n])(?=.*[A-Z])(?=.*[a-z]).*$" required="required" />
+                </label>
+            </div>
+           <div class="row">
+             <button type="submit">sign up</button>
+           </div>
+        </form>
+    </div>
+</div>
+<!--end of signup form popup-->
```
[the remark this answers is not in the document]
Comment (minhvip1997) [reply rikai/resources/views/users/layouts/header.blade.php]: em đã fix

File: rikai/resources/views/users/layouts/header.blade.php [hunk 33]
```diff
@@ -0,0 +1,202 @@
...
+								<li class="it-last"><a href="comingsoon.html">Coming soon</a></li>
+							</ul>
+						</li>                
+						<li><a href="#">Help</a></li>
+						<li class="loginLink"><a href="#">LOG In</a></li>
+						<li class="btn signupLink"><a href="#">sign up</a></li>
+					</ul>
+				</div>
+			<!-- /.navbar-collapse -->
+	    </nav>
```
[the remark this answers is not in the document]
Comment (minhvip1997) [reply rikai/resources/views/users/layouts/header.blade.php]: em đã fix

File: rikai/resources/views/users/profile/index.blade.php (hunk 17 above)
Comment (minhvip1997) [reply rikai/resources/views/users/profile/index.blade.php:112]: em đã fix

File: rikai/resources/views/users/layouts/slide.blade.php (hunk 16 above)
Comment (minhvip1997) [reply rikai/resources/views/users/layouts/slide.blade.php]: em đã fix

File: rikai/resources/views/users/layouts/index.blade.php (hunk 15 above)
Comment (minhvip1997) [reply rikai/resources/views/users/layouts/index.blade.php:12]: em đã fix

File: rikai/resources/views/users/review/show.blade.php (hunk 21 above)
Comment (minhvip1997) [reply rikai/resources/views/users/review/show.blade.php:107]: em đã fix

File: rikai/resources/views/users/review/edit.blade.php (hunk 20 above)
Comment (minhvip1997) [reply rikai/resources/views/users/review/edit.blade.php]: em đã fix

File: rikai/resources/views/users/review/create.blade.php (hunk 19 above)
Comment (minhvip1997) [reply rikai/resources/views/users/review/create.blade.php:89]: em đã fix

File: rikai/resources/views/users/review/create.blade.php (hunk 18 above)
Comment (minhvip1997) [reply rikai/resources/views/users/review/create.blade.php:60]: em đã fix

File: rikai/routes/web.php (hunk 22 above)
Comment (minhvip1997) [reply rikai/routes/web.php]: em đã fix

File: rikai/resources/views/users/book/list.blade.php [hunk 34]
```diff
@@ -0,0 +1,282 @@
...
+						<option value="popularity">Popularity Descending</option>
+						<option value="popularity">Popularity Ascending</option>
+						<option value="rating">Rating Descending</option>
+						<option value="rating">Rating Ascending</option>
+						<option value="date">Release date Descending</option>
+						<option value="date">Release date Ascending</option>
+					</select>
+					<a href="movielist.html" class="list"><i class="ion-ios-list-outline "></i></a>
+					<a  href="moviegridfw.html" class="grid"><i class="ion-grid active"></i></a>
+				</div>
```
[the remark this answers is not in the document]
Comment (minhvip1997) [reply rikai/resources/views/users/book/list.blade.php]: cái tính năng nó dư ra em dùng nữa nên xóa rồi ạ

**Review by namtprikai — APPROVED** [review]

**Pull request merged**